src/libpgmoneta/walfile/wal_reader.c

```diff
@@ -279,14 +308,17 @@ pgmoneta_wal_parse_wal_file(char* path, struct server* server_info, struct walfi
 
       decoded = calloc(1, sizeof(struct decoded_xlog_record));
 
-      if (pgmoneta_wal_decode_xlog_record(buffer, decoded, record, long_header->xlp_xlog_blcksz, server_info))
+      if (pgmoneta_wal_decode_xlog_record(buffer, decoded, record, long_header->xlp_xlog_blcksz, long_header->std.xlp_magic))
       {
          pgmoneta_log_fatal("error in decoding\n");
```

Comment:
Here is one

Reply:
They are in src/libpgmoneta/walfile.c and src/libpgmoneta/walfile/wal_reader.c

Reply:
sorry sorry I don't understand this and the one above, can you please elaborate?

Reply:
ah I see, you mean \n in the pgmoneta_logs, fixed!